extension/textext/__init__.py

```diff
@@ -651,6 +628,9 @@ def _svg_to_textext_node(self, svg_filename):
 
             self.make_ids_unique()
 
+            # Ensure that snippet is correctly scaled according to the units of the document
+            self.transform.add_scale(doc_unit_to_mm/root.unittouu("1mm"))
```

Comment:
We want to scale by ratio of doc units to root units. It seems like we should scale by inverse of what written.
```python 
doc/root = (doc/mm) / (root/mm) = ( mm / root ) / ( mm / doc) = mm_to_root / mm_to_doc
```

So it should be
```
self.transform.add_scale(root.unittouu("1mm")/mm_to_doc)
```
, right?

Reply:
No, the other way round as in the commit. ``root.unittouu("1mm")/mm_to_doc`` leads to completely wrong scaling (much too large).

Reply:
Found where I was mistaken. `svg.unittouu("1mm")` returns "user_units_per_mm", not 
"mm_per_user_unit".

Reply:
I suggest to rename `mm_to_doc` to less confusing `user_units_per_mm`.